src/content/zh-tw/updates/2017/10/remove-shadow-piercing.md

```diff
@@ -0,0 +1,42 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: 向shadow-piercing CSS選擇器說再見。
+
+{# wf_updated_on: 2017-10-24 #}
+{# wf_published_on: 2017-10-24 #}
+{# wf_tags: webcomponents,shadowdom,style,css,deprecations,removals #}
+{# wf_featured_image: /web/updates/images/generic/styles.png #}
+{# wf_featured_snippet: 向shadow-piercing CSS選擇器說再見。 #}
+{# wf_blink_components: Blink>DOM #}
+
+# Chrome 63將刪除::shadow 和 /deep/ {: .page-title }
+
+{% include "web/_shared/contributors/robdodson.html" %}
+
+Translated by
+{% include "web/_shared/contributors/henrylim.html" %}
+
+在Chrome63之後，你將無法使用shadow-piercing 選擇器， `::shadow` 和 `/deep/` 來設計(style) shadow root裡面的內容。
+
+- `/deep/` 组合者(combinator)將會成為後代選擇器。 `x-foo /deep/ div` 將會當成 `x-foo div`.
+- `::shadow` 偽元素(pseudo-element)將不會匹配任何元素。
+
+注：如果你的網頁使用Polymer, Polymer團隊已發布了一個[如何讓你遷移](https://www.polymer-project.org/blog/2017-10-18-upcoming-changes.html) `::shadow` 和 `/deep/`的文章。
```

Comment:
[Callouts](https://developers.google.com/web/resources/markdown-syntax#callouts) like `Note:` should NOT be translated, they're handled differently by the markdown processor.